PWGHF/D2H/TableProducer/dataCreatorCharmResoReduced.cxx

```diff
@@ -992,7 +992,7 @@ struct HfDataCreatorCharmResoReduced {
     }
     registry.fill(HIST("hEvents"), 1 + Event::DV0Selected);
     float centrality = -1.f;
-    uint16_t hfRejMap = hfEvSel.getHfCollisionRejectionMask<true, o2::hf_centrality::CentralityEstimator::None, aod::BCsWithTimestamps>(collision, centrality, ccdb, registry);
+    uint32_t hfRejMap = hfEvSel.getHfCollisionRejectionMask<true, o2::hf_centrality::CentralityEstimator::None, aod::BCsWithTimestamps>(collision, centrality, ccdb, registry);
```

Comment:
This variable is stored in the tables produced by these data producers (which are written on disk): https://github.com/AliceO2Group/O2Physics/blob/a2a96905974a13d9c77cb15b0f3f2eeac487e3f2/PWGHF/D2H/DataModel/ReducedDataModel.h#L44 
This means that the definition of the column should also change and a converter should be implemented in order to be able to analyse the already produced derived data (otherwise this change will break the analysis of already produced derived data of B mesons and Ds resonances)